src/google/protobuf/compiler/csharp/csharp_names.h

```diff
@@ -90,9 +92,7 @@ string GetReflectionClassName(const FileDescriptor* descriptor);
 //    will contain the error message.
```

Comment:
I think this *might* break GRPC; @jtattermusch, are you using this function? (It might be simpler just to extract the relevant options when calling this.)